core/src/consensus/validator_set/dynamic_validator.rs

```diff
@@ -99,17 +67,14 @@ impl DynamicValidator {
         }
```

Comment:
Shouldn't we remove this?

Reply:
Yes, we should remove it.
I'll do it and remove PreviousValidators stored in the state in another PR.

Reply:
I added the commit removing the code related to the previous validators.
I planed to make the other PR because actually it should be done by https://github.com/CodeChain-io/foundry/pull/277, but I think adding the commit in this PR also seems good.